moveit_setup_assistant/include/moveit/setup_assistant/tools/moveit_config_data.h

```diff
@@ -157,7 +157,11 @@ class MoveItConfigData
   bool urdf_from_xacro_;
 
   /// URDF robot model
+#if __cplusplus <= 199711L
   boost::shared_ptr<urdf::Model> urdf_model_;
+#else
+  std::shared_ptr<urdf::Model> urdf_model_;
+#endif
```

Comment:
Actually, it looks this file isn't really part of the public API and it is never passed to a `urdfdom` function either. In that case the exact `shared_ptr` type doesn't matter too much I think. Maybe it could even stay a `boost::shared_ptr`?

It is passed to `robot_model::RobotModel`, which currently still wants a `boost::shared_ptr`. And updating that does change the public API. That makes it a bit tricky..